velox/exec/prefixsort/PrefixSortEncoder.h

```diff
@@ -67,7 +68,8 @@ class PrefixSortEncoder {
   /// @return For supported types, returns the encoded size, assume nullable.
   ///         For not supported types, returns 'std::nullopt'.
   FOLLY_ALWAYS_INLINE static std::optional<uint32_t> encodedSize(
-      TypeKind typeKind) {
+      TypeKind typeKind,
+      std::optional<uint32_t> stringPrefixLength) {
```

Comment:
Shall we always require pass a value here, and it could be from the value from prefix config if not set from stats?
```
s/stringPrefixLength/maxStringPrefixLength/
```